Dockerfile.simpleci

```diff
@@ -24,17 +25,14 @@ RUN echo 'Defaults    secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr
 RUN mkdir -p /home/ci/.cache
 
 RUN groupadd -g 999 ci && \
-    useradd -r -u 999 -g ci ci
-
-RUN chown -R ci:ci /go
-RUN chown -R ci:ci /home/ci
+    useradd -r -u 999 -g ci ci && \
+    chown -R ci:ci /go /home/ci
 
 USER ci
 
 RUN for d in $(go list ./... | grep -v browser); do go test -v -race --timeout 15m "$d"; done
 RUN make verifiers
 RUN make crosscompile
-RUN make coverage || true
```

Comment:
we just need to remove the catch all fail-safe

Reply:
yeah, I removed coverage because it was useful previously to report this value to codecov.io - through SimpleCI if we need to report we need to add one more tool to report to CI the code-coverage.

Reply:
gotcha